capstone-backend/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,30 @@
+package com.google.sps.data;
+
+
+public class Comment extends BaseEntity {
+  private final String text;
+  private final String userID;
+  private final String whenCreated;
+
+  public Comment(String text, String userID, String whenCreated) {
+    this.text = text;
+    this.userID = userID;
+    this.whenCreated = whenCreated; 
```

Comment:
Will this Comment object be use to store comments on Firestore? Don't forget to take a look into Firebase Messaging API.

- https://firebase.google.com/docs/cloud-messaging/js/client
- https://firebase.google.com/docs/cloud-messaging/js/receive
- https://www.youtube.com/watch?v=BsCBCudx58g

Reply:
I wanted to discuss this with the team, as we try to figure out what we should fit into our MVP and what is not core to our application. I looked into the API, and we can discuss as a team on Monday.